pulp_rpm/app/tasks/utils.py

```diff
@@ -0,0 +1,213 @@
+import requests
+from urllib.parse import urljoin
+
+from productmd.common import SortedConfigParser
+from productmd.treeinfo import TreeInfo
+
+
+def url_exists(url):
+    """
+    Check if URL exists.
+
+    """
+    response = requests.get(url)
+    return response.status_code == 200
+
+
+def get_kickstart_data(remote):
+    """
+    Get Kickstart data from remote.
+
+    """
+    kickstart = None
+    namespaces = [".treeinfo", "treeinfo"]
+    for namespace in namespaces:
+        response = requests.get(urljoin(remote.url, namespace))
```

Comment:
@bmbouter since I moved it out of the `FirstStage`, I couldn't use the `downloader` since it was `async`. For now I'm using `requests` do you think we can find a better way to replace it?